bom/runtime/pom.xml

```diff
@@ -173,9 +173,9 @@
         <sundr.version>0.19.1</sundr.version> <!-- this is to avoid annoying pop-up in eclipse about failure to init Velocity logging -->
         <flapdoodle.mongo.version>2.2.0</flapdoodle.mongo.version>
         <wiremock.version>2.24.1</wiremock.version>
-        <quarkus-spring-api.version>5.2.Final</quarkus-spring-api.version>
+        <quarkus-spring-api.version>5.3.Alpha1-SNAPSHOT</quarkus-spring-api.version>
         <quarkus-spring-data-api.version>2.1.Final</quarkus-spring-data-api.version>
-        <quarkus-spring-security-api.version>5.2.Final</quarkus-spring-security-api.version>
+        <quarkus-spring-security-api.version>5.3.Alpha1-SNAPSHOT</quarkus-spring-security-api.version>
```

Comment:
Spring Security has a 5.3 but did you upgrade to it?

Reply:
No, this was just changed by mistake

Reply:
Fixed